buildpipeline/pipelinejobs.groovy

```diff
@@ -0,0 +1,55 @@
+// Import the utility functionality.
+
+import jobs.generation.JobReport;
+import jobs.generation.Utilities;
+import org.dotnet.ci.pipelines.Pipeline
+
+// The input project name (e.g. dotnet/core-setup)
+def project = GithubProject
+// The input branch name (e.g. master)
+def branch = GithubBranchName
+
+// **************************
+// Define innerloop testing. Any configuration in ForPR will run for every PR but all other configurations
+// will have a trigger that can be
+// **************************
+
+def alpine36Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/alpine.3.6.groovy')
+
+def configurations = [
+    ['TGroup':"netcoreapp", 'Pipeline':alpine36Pipeline, 'Name':'Alpine.3.6' ,'ForPR':"", 'Arch':['x64']],
+]
+
+configurations.each { config ->
+ ['Debug', 'Release'].each { configurationGroup ->
+  (config.Arch ?: ['x64', 'x86']).each { archGroup ->
+    def triggerName = "${config.Name} ${archGroup} ${configurationGroup} Build"
+
+    def pipeline = config.Pipeline
+    def params = ['TGroup':config.TGroup,
```

Comment:
I think a good number of these parameters are unused in here.  It looks like only CGroup is used in the buld commands?

Reply:
I've used the same arguments as we used in corefx. But now I can see that the TGroup and TestOuter are not needed for core-setup, so I'll remove them. I've updated the alpine groovy file to use the AGroup instead of explicit x64 to be future proof. We will need to AGroup to be here anyways after we merge all the CI jobs to pipelines.